apparmor.d/groups/kde/kded5

```diff
@@ -14,6 +14,7 @@ profile kded5 @{exec_path} {
   include <abstractions/bus/org.bluez>
   include <abstractions/consoles>
   include <abstractions/dconf-write>
+  include <abstractions/dbus-network-manager-strict>
```

Comment:
Only use dbus abs from this project (in `bus/`)

Reply:
I'll come back to this after some testing. I don't remember which dbus rules exactly are required, so I'll run it without this include first.

Reply:
Most likelly you need: `include <abstractions/bus/org.freedesktop.NetworkManager>`